fv/winfv/policy_test.go

```diff
@@ -117,5 +131,34 @@ var _ = Describe("Windows policy test", func() {
 			err := kubectlExec(`-t porter -- powershell -Command 'Invoke-WebRequest -UseBasicParsing -TimeoutSec 5 www.google.com'`)
 			Expect(err).To(HaveOccurred())
 		})
+		It("porter pod can connect to kube apiserver after creating service egress policy", func() {
+			// Assert API is not reachable.
+			err := kubectlExec(`-t porter -- powershell -Command 'Invoke-WebRequest -UseBasicParsing -TimeoutSec 5 https://kubernetes.default'`)
```

Comment:
I think you should use `https://kubernetes.default.svc.cluster.local` as target (same as the connection made below). Because if there is an DNS issue with short format of service name, the test can still pass with or without applying service policy.